pyvista/core/utilities/geometric_sources.py

```diff
@@ -2193,17 +2193,37 @@ class PlaneSource(_vtk.vtkPlaneSource):
     j_resolution : int, default: 10
         Number of points on the plane in the j direction.
 
+    center : sequence[float], default: (0.0, 0.0, 0.0)
+        Center in ``[x, y, z]``.
+
+    origin : sequence[float], default: (-0.5, -0.5, 0.0)
+        Origin in ``[x, y, z]``.
+
+    pointa : sequence[float], default: (0.5, -0.5, 0.0)
```

Comment:
This may be a style thing, but I prefer having the underscore, and matches the style of `i_resolution` and `j_resolution`

```suggestion
    point_a : sequence[float], default: (0.5, -0.5, 0.0)
```